ui/app/AppLayouts/Chat/panels/communities/CommunityPermissionsSettingsPanel.qml

```diff
@@ -0,0 +1,155 @@
+import QtQuick 2.14
+import QtQuick.Layouts 1.14
+import QtQuick.Controls 2.14
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Components 0.1
+import StatusQ.Popups 0.1
+
+import utils 1.0
+import shared.controls.chat 1.0
+
+import "../../layouts"
+
+SettingsPageLayout {
+    id: root
+
+    signal addPermission()
+
+    title: qsTr("Permissions")
+
+    content: Item {
```

Comment:
Have you considered what should happen when content is too big for the viewport? I would wrap it into flickable at least :thinking: